test/spec/services/hpaServiceSpec.js

```diff
@@ -22,8 +21,7 @@ describe('HPAService', function() {
   }));
 
 
-  beforeEach(inject(function(_$q_, _$rootScope_, _$window_, _HPAService_) {
-    $rootScope = _$rootScope_;
+  beforeEach(inject(function(_$q_, _$window_, _HPAService_) {
```

Comment:
@spadgett also did remove the `$rootScope` `$digest`.  Thought that was necessary to get `$q` to resolve in this env, but guess not!